pipeline.yaml

```diff
@@ -613,7 +613,8 @@ deploymentSpec:
           && cd {{$.outputs.artifacts[''taxonomy''].path}} && if [ -n "{{$.inputs.parameters[''repo_branch'']}}"
           ]; then git fetch origin {{$.inputs.parameters[''repo_branch'']}} && git
           checkout {{$.inputs.parameters[''repo_branch'']}}; elif [ -n "{{$.inputs.parameters[''repo_pr'']}}"
-          && {{$.inputs.parameters[''repo_pr'']}} -gt 0 ]; then git fetch origin pull/{{$.inputs.parameters[''repo_pr'']}}/head:{{$.inputs.parameters[''repo_pr'']}}
+          ] && [ {{$.inputs.parameters[''repo_pr'']}} -gt 0 ]; then git fetch origin
+          pull/{{$.inputs.parameters[''repo_pr'']}}/head:{{$.inputs.parameters[''repo_pr'']}}
```

Comment:
I don't understand why these lines changed, any idea? `git_clone_op` was not modified.

Reply:
I think it's my mistake. [In this PR](https://github.com/redhat-et/ilab-on-ocp/pull/44), I have forgot to generate the pipeline.yaml after this change.

Reply:
Ok, thanks for the clarification.

Reply:
@JudeNiroshan  can you remove this change from this PR? Then we can merge it 😃

Reply:
I think we should have https://github.com/redhat-et/ilab-on-ocp/pull/55 first, to fix the pipeline.

Reply:
Thanks, @JudeNiroshan please rebase.